modules/issue_tracker/test/Issue_TrackerTest.php

```diff
@@ -100,7 +100,7 @@ function testIssueTrackerDoespageLoad()
      */
```

Comment:
line :63 
         $this->DB->insert(
             "issues",
             array(
              'issueID'  => '999999',
              'title'    => 'Test Issue',
              'status'   => 'new',
              'priority' => 'low',
              'reporter' => 'TestUser',
              'centerID' => '55',     // add centerID here
              )